packages/wonder-blocks-clickable/package.json

```diff
@@ -21,12 +21,12 @@
     "aphrodite": "^1.2.5",
     "popper.js": "^1.14.1",
```

Comment:
Nit: This is out of scope for this PR, but I just noticed that we are including `popper.js` here even that Clickable don't use it.